pkg/orchestration/docker.go

```diff
@@ -114,7 +114,11 @@ func runNode(
 	}
 	httpPorts := fmt.Sprintf("%d:%d", port, port)
 	httpsPorts := fmt.Sprintf("%d:%d", tlsPort, tlsPort)
-	portSet, portBindings, err := nat.ParsePortSpecs([]string{httpPorts, httpsPorts})
+	allPorts := []string{httpPorts, httpsPorts}
+	if config.HostPorts != "" {
+		allPorts = append(allPorts, strings.Split(config.HostPorts, ",")...)
+	}
+	portSet, portBindings, err := nat.ParsePortSpecs(allPorts)
```

Comment:
We could open these ports on a bridge network instead of adding them to the host network. That way navicat can connect via ssh and then from the host machine connect to a port on the dind container, rather than exposing the port directly on the host.

I was having trouble testing this when I tried it last week but I think your other PR may have fixed those issues. What are your thoughts?